sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 1.0.1 (Unreleased)
 
+- The `text` property of the `SentenceSentiment` interface is no longer optional because the service always returns it. This interface is implemented by output types, so this change does not break existing code that calls the library APIs.
```

Comment:
```suggestion
- The `text` property of the `SentenceSentiment` interface is no longer optional because the service always returns it. This interface is used exclusively as an output type so this change does not break existing code.
```